README.md

```diff
@@ -14,8 +14,12 @@ Supported clouds are:
 To see available options run ./scripts/cdperf --help
 
 ```shell
+$ # build via docker
 $ make clean build
 $
+$ # alternatively, build locally (requires go, k6, node.js + yarn installed)
```

Comment:
only node/yarn is needed to build it, then running it locally would need go && k6 to be installed

Reply:
Yes, you are correct, I will make this clearer in the docs.